crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -1601,6 +1728,20 @@ mod tests {
         );
     }
 
+    #[test]
+    fn rhombus_closest_point() {
+        let rhombus = Rhombus::new(2.0, 1.0);
```

Comment:
For completeness, we should add a test for a null rhombus (0.0, 0.0)

Reply:
Yes, that wasn't tested, so we added the test in our latest commit